### DISCUSSION TITLE
.Image() existed png file - strange effect, need help

### DISCUSSION THREAD

**Discussion by vihrovart** [QuestPDF/QuestPDF#805] [Q&A] [answered]
Hi there!

I'm trying to place image (existed png file), but every time I have got same result - image in generated pdf is broken.
Like this
<img width="565" alt="image" src="https://github.com/QuestPDF/QuestPDF/assets/44255644/e68d223e-c7c7-4cd1-96e6-4afb4ef3354c">

I have tried stream, filePath, byteArray, same result.

My code
`private void ComposeContent(IContainer container)
    {
        container
            .PaddingVertical(20)
            .Column(
            column =>
            {
                using var fileStream = new FileStream("/app/local_storage/google.png", FileMode.Open, FileAccess.Read);
                column.Item().Image(fileStream);
            });
    }`

If I trying to save this stream (fileStream) to file, it is normal (good) image. And I tried many images (files) from different places (internet, generated by my self, screenshots and so one).

Maybe some one know what the problem?

**Comment by todd-banister** [comment, marked as answer]
Since you have the squiggly rectangle in the upper left hand corner of the image, you might be running into the same thing I did in this [thread](https://github.com/QuestPDF/QuestPDF/discussions/738).

**Reply by MarcinZiabek** [reply]
As far as I know, SkiaSharp 2.88.7 does fix the issue. Does it also work for you?

**Reply by vihrovart** [reply]
Yes, problem solved!
Required dependencies in start up project:
<img width="319" alt="image" src="https://github.com/QuestPDF/QuestPDF/assets/44255644/f211be11-20ca-4928-b029-16442fcfd407">
<img width="381" alt="image" src="https://github.com/QuestPDF/QuestPDF/assets/44255644/9a4220c8-b8bf-40d1-ac17-4f3b9b6e5507">
(Mac, Docker linux, Rider, .net 7)

**Comment by DawidRobertNowak** [comment]
Hello,

We are closing this thread due to inactivity. Thank you for taking the time to reach out and contribute to the project — your feedback helps us improve QuestPDF.

Should you encounter any further problems, please don't hesitate to open a new issue.

Thank you!
